lib/global-styles.php

```diff
@@ -53,15 +53,15 @@ function_exists( 'gutenberg_is_edit_site_page' ) &&
 
 		$new_global_styles = array();
 
-		$css_variables = wp_get_global_stylesheet( array( 'variables' ) );
+		$css_variables = gutenberg_get_global_stylesheet( array( 'variables' ) );
```

Comment:
I'm not entirely convinced that a rename here is necessary for the changes in this PR, but I thought it'd be safer to port them.

Reply:
Actually, they are :) See https://github.com/WordPress/gutenberg/pull/38883